aloscene/frame.py

```diff
@@ -507,10 +510,27 @@ def mean_std_norm(self, mean, std, name) -> Frame:
 
         return tensor
 
+    def norm_meanstd(self, mean_std=None, name=None) -> Frame:
```

Comment:
Having two methodes with nearly the same name is not a good practice. try to fix `mean_std_norm` instead